client/state/reader/start/selectors.js

```diff
@@ -21,7 +31,9 @@ export function isRequestingRecommendations( state ) {
  * @return {Object} Recommendation
  */
 export function getRecommendationById( state, recommendationId ) {
-	return state.reader.start.items[ recommendationId ];
+	return find( state.reader.start.items, ( item ) => {
```

Comment:
you can use the property shorthand here to shorten up the code a bit. `return find( state.reader.start.items, { ID: recommendationId } )`